pkg/forwarder/forwarder.go

```diff
@@ -20,23 +20,50 @@ import (
 )
 
 var (
-	forwarderExpvar        = expvar.NewMap("forwarder")
-	transactionsExpvar     = expvar.Map{}
-	retryQueueSize         = expvar.Int{}
-	successfulTransactions = expvar.Int{}
-	droppedOnInput         = expvar.Int{}
-	apiKeyStatus           = expvar.Map{}
+	forwarderExpvars           = expvar.NewMap("forwarder")
+	transactionsExpvars        = expvar.Map{}
```

Comment:
Could you move the `expvar.Int` only used in the `domainForwarder` or `Trasanction` to the domain_forwarder.go or transaction.go ?